pages/common/ss-local.md

```diff
@@ -0,0 +1,20 @@
+# ss-local
+
+> Shadowsocks client as SOCKS5 proxy.
+> More information: <https://github.com/shadowsocks/shadowsocks-libev/blob/master/doc/ss-local.asciidoc>.
+
+- Run Shadowsocks proxy by specifying host, server port, local port, password, and encrypt method:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a Shadowsocks proxy by specifying the host, server port, local port, password, and encryption method:
```